src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
-  external json: t => 'a => unit = "" [@@bs.send];
+  external sendString : t => string => unit = "send" [@@bs.send];
+  external sendObject : t => Js.t 'a => unit = "send" [@@bs.send];
```

Comment:
faster than I can keep up with. I guess I already had the latest version because it works for me without updating!